app/assets/stylesheets/openproject/_homescreen.sass

```diff
@@ -28,9 +28,10 @@
 
 @import openproject/mixins
 
-.controller-homescreen,
-.controller-homescreen #content
-  background: $widget-box-content-bg-color
+.controller-homescreen
+  #content-wrapper,
+  #content
+    background: $grid-background-color
```

Comment:
It is new for me that the changed background should also be applied to the start page. It probably makes sense, because otherwise you won't see the widgets well. However, there are a few other things to consider. For example, the footer doesn't stand out anymore. And some of the changes to the distances don't apply to the home page, which makes them look different again. At least the footer should be better visible again.

![Bildschirmfoto 2019-10-14 um 09 34 14](https://user-images.githubusercontent.com/7457313/66737105-5cf54500-ee6b-11e9-9c47-cc166f42d803.png)

Reply:
The my page should also receive the gray background.

![image](https://user-images.githubusercontent.com/617519/66746647-b026c200-ee82-11e9-814c-d10a56a7281a.png)

Reply:
👍